Cargo.toml

```diff
@@ -1,7 +1,7 @@
 [package]
 
 name    = "inotify"
-version = "0.9.1"
+version = "0.9.2"
```

Comment:
Can you please remove this change? I will update the version when I do the release.

Reply:
Sure.